nemo/core/neural_types/elements.py

```diff
@@ -140,49 +152,93 @@ class ChannelType(ElementType):
     """Element to represent convolutional input/output channel.
     """
 
+    def __init__(self):
+        """Dummy init for TorchScript compatibility"""
```

Comment:
These classes should call super not pass

Reply:
See comment about `super` above. I can change everywhere, if you think it's a better solution: 

```python
def __init__(self):
    """Dummy init for TorchScript compatibility"""
    if not torch.jit.is_scripting():
        # torch.jit does not support super() call
        super().__init__()
```

Reply:
Yes this is better

Reply:
Done